kn/kn_deployer.yaml

```diff
@@ -7,8 +7,9 @@ metadata:
   namespace: tkn-kn
 # Link your docker registry secrets
 secrets:
-  - name: dockerreg
-
+  - name: container-reg
+imagePullSecrets:
+  - name: container-reg
```

Comment:
Should this be `container-registry` ? That's the `docker-registry` secret you define above.

Reply:
Yup

Reply:
Fixed here! 
https://github.com/navidshaikh/tekton-pipelines/commit/fa13d54c57df8b1ca50aab671dac4216b67e9562

Thanks for review Martin!